src/documentation/pages/components/forms/search-input/examples/_portal_sizes.vue

```diff
@@ -0,0 +1,137 @@
+<template lang="pug">
+  <ComponentExample title="Sizes" id="sizes-portal" :tabs="exampleTabs">
+    div(slot="example-description")
+      p.-text Search input supports the following sizes: <code>xs</code>, <code>sm</code>, <code>md</code>,
+        |  <code>lg</code>.
+        | The default size is <code>md</code>.
+    .-p--0(slot="example")
+      .chi-grid
+      each size in ['xs', 'sm', 'md', 'lg']
```

Comment:
We should use v-for loops here, something like that. 
```suggestion
      .chi-grid(v-for="size in ['xs', 'sm', 'md', 'lg', 'xl']")
```
You can take a look in other examples.

Reply:
same here